megalinter/descriptors/kubernetes.megalinter-descriptor.yml

```diff
@@ -70,3 +70,28 @@ linters:
               && mv ${ML_THIRD_PARTY_DIR}/kubeconform /usr/local/bin \
               && rm ${ML_THIRD_PARTY_DIR}/kubeconform-linux-amd64.tar.gz \
               && find ${ML_THIRD_PARTY_DIR} -type f -not -name 'LICENSE*' -delete -o -type d -empty -delete
+
+  # HELM LINT
+  - linter_name: helm
+    name: KUBERNETES_HELM
+    linter_repo: https://github.com/helm/helm
+    linter_url: https://helm.sh/docs/helm/helm_lint/
+    linter_version_cache: "v3.11.1"
```

Comment:
Why not using `cli_version_arg_name: version` ?
With a hardcoded version, when there will be a new one we won't see it
You may need to define `version_extract_regex` if the default semver regex does not work

Reply:
feat: use cli_version_arg_name for helm linter (autoupdate version)

I leave it to you to resolve this conversation if you wish.